load_tests/load_test.py

```diff
@@ -411,27 +408,22 @@ def publish_fluent_config_s3(session, input_logger):
     )
     return f'arn:aws:s3:::{bucket_name}/{OUTPUT_PLUGIN}-test/{PLATFORM}/fluent-{input_logger["name"]}.conf'
 
-# The following method is used to clear data between
-# testing batches
+# The following method is used to clear data after all tests run
 def delete_testing_data(session):
-    # All testing data related to the plugin option will be deleted
-    if OUTPUT_PLUGIN == 'cloudwatch':
```

Comment:
the "delete_testing_resources" only runs once, so its no longer necessary to check the plugin here. The delete happens just once at the end of the cloudwatch tests in the main "delete_testing_resources" routine.